module/VuFind/src/VuFind/Controller/CartController.php

```diff
@@ -55,16 +57,25 @@ class CartController extends AbstractBase
      */
     protected $session;
 
+    /**
+     * VuFind configuration
+     *
+     * @param Config
+     */
+    protected $config;
+
     /**
      * Constructor
      *
      * @param ServiceLocatorInterface $sm        Service manager
      * @param Container               $container Session container
+     * @param Config                  $config    VuFind configuration
      */
-    public function __construct(ServiceLocatorInterface $sm, Container $container)
+    public function __construct(ServiceLocatorInterface $sm, Container $container, Config $config)
     {
         parent::__construct($sm);
         $this->session = $container;
+        $this->config = $config;
```

Comment:
Might be better to pass the Config Plugin Manager here so we can access multiple configurations (i.e. if you want to take my suggestion about moving settings to export.ini).